build.sbt

```diff
@@ -725,7 +733,8 @@ lazy val benchmark =
         // For ScalaPB
         // "com.thesamet.scalapb" %% "scalapb-runtime-grpc" % scalapb.compiler.Version.scalapbVersion
         // For grpc-java
-        "io.grpc" % "grpc-protobuf" % GRPC_VERSION
+        "io.grpc"             % "grpc-protobuf" % GRPC_VERSION,
+        "com.google.protobuf" % "protobuf-java" % "3.11.0"
```

Comment:
Because of this missing dependency, 20.9.0 release was failed once so I needed to manually publish 20.9.0 artifacts.

Reply:
Oh, that must have been tough. Thank you